smart_open/bytebuffer.py

```diff
@@ -159,3 +159,22 @@ def fill(self, source, size=-1):
 
         self._bytes += new_bytes
         return len(new_bytes)
+
+    def readline(self, terminator):
+        """Efficiently reads a line from this buffer.
```

Comment:
```suggestion
        """Read a line from this buffer efficiently.
```

(docstrings in imperative mode)